components/01-globals/variables/_variables.scss

```diff
@@ -1,15 +1,15 @@
 // colors
 
-$color-main1: #f26322;
-$color-main2: #ff5000;
-$color-main3: #883008;
+$color-orange: #f26322;
```

Comment:
Still nope. Do not describe content of the variable, b/c it may change, so `#color-orange` may contain a blue color 😉 

It should be always description of possible use case or role, i.e. `color-primary`, `color-background`, `color-accent` etc.